language-plutus-core/src/Language/PlutusCore.hs

```diff
@@ -141,47 +143,54 @@ fileType :: FilePath -> IO T.Text
 fileType = fileNormalizeType False
 
 fileNormalizeType :: Bool -> FilePath -> IO T.Text
-fileNormalizeType norm = fmap (either prettyPlcDefText id . printNormalizeType norm) . BSL.readFile
+fileNormalizeType norm = fmap (either @(Error AlexPosn) prettyPlcDefText id . printNormalizeType norm) . BSL.readFile
```

Comment:
This should just be a type signature. No need for type applications here.

Reply:
You really don't like type applications, do you? :) I find it nicer than lifting random subexpressions, but I'll do that if you prefer.

Reply:
Hmm, I do like the type application TBH. What is bad about it?

Reply:
I have nothing against type applications as well.